db/migrate/20180828160700_add_body_to_push_notification.rb

```diff
@@ -0,0 +1,10 @@
+class AddBodyToPushNotification < ActiveRecord::Migration
+  def up
+    add_column :push_notifications, :body, :string, null: false, default: 'Body for existing records'
+    change_column_default(:push_notifications, :body, nil)
```

Comment:
Why this? We already set a default in the previous line, also we have a not null constrain.